### ISSUE TITLE
Nameless commands triggering NullReferenceException on AddModulesAsync

### ISSUE THREAD

**Issue by james7132** [RogueException/Discord.Net#365]
Setup:
```csharp
[Group("test")]
public class TestModule : ModuleBase {
  [Command]
  public async Task TestCommand() {
    ...
  }
}
```
Result of using ``CommandService.AddModulesAsync``:
```
Unhandled Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at Discord.Commands.CommandMapNode.NextWhitespace(String text, Int32 startIndex)
   at Discord.Commands.CommandMapNode.AddCommand(String text, Int32 index, CommandInfo command)
   at Discord.Commands.CommandMap.AddCommand(CommandInfo command)
   at Discord.Commands.CommandService.LoadModuleInternal(ModuleInfo module)
   at Discord.Commands.CommandService.LoadModuleInternal(ModuleInfo module)
   at Discord.Commands.CommandService.<AddModulesAsync>d__12.MoveNext()
```

**Comment by FiniteReality** [comment]
Mind running a debugger and seeing what the command's aliases are? This may be a result of my command builders code or volt's reworked command map - if we know what the command's aliases are it'll be easier to narrow down.

**Comment by james7132** [comment]
The command only has one element under alias, and it is indeed null from the null initialization in CommandAttribute.

**Comment by FiniteReality** [comment]
Okay, that means it's a problem on my side. I'll fix it.

**Comment by james7132** [comment]
Fixed as of  #367
